tests/lean/run/list_simp.lean

```diff
@@ -125,9 +125,9 @@ variable (w : l₁ ≠ []) in
 
 /-! ### concat -/
 
-/-! ### join -/
+/-! ### flatten -/
 
-#check_simp (L₁ ++ L₂).join ~> L₁.join ++ L₂.join
```

Comment:
Was this intended for a different PR?

Reply:
There were deprecation warnings in this file, that CI doesn't complain about, so I did a drive by clean up while I was in the file.